sky/check.py

```diff
@@ -61,4 +62,12 @@ def get_cloud_credential_file_mounts() -> Dict[str, str]:
     for cloud in enabled_clouds:
         cloud_file_mounts = cloud.get_credential_file_mounts()
         file_mounts.update(cloud_file_mounts)
+    # Currently, get_enabled_clouds() does not support r2
+    # as only clouds with computing instances are supported
+    # by 'clouds'
```

Comment:
```suggestion
    # Currently, get_enabled_clouds() does not support r2
    # as only clouds with computing instances are marked
    # as enabled by skypilot. This will be removed when
    # cloudflare/r2 is added as a 'cloud'.
```